shared/stats/comment.js

```diff
@@ -0,0 +1,68 @@
+/**
+ *
+ * @param {object} githubContext - Github Action context
+ * @param {import('@octokit/rest').Octokit} githubContext.github
+ * @param {import('@actions/github').context} githubContext.context
+ * @param {string} markerText - A unique marker used to identify the correct comment
+ * @param {string} bodyText - The body of the comment
+ * @returns {Promise<object>} Github request object
+ */
+const commentOnPr = async function ({ github, context }, markerText, bodyText) {
```

Comment:
Super tiny thing but named functions will appear in stack traces

```suggestion
export async function commentOnPr ({ github, context }, markerText, bodyText) {
```

Reply:
Using `export` here (versus `module.exports` later) assumes we'd rename to `*.mjs` too